[
  {
    "url": "https://api.github.com/repos/grpc/grpc/issues/comments/192134971",
    "html_url": "https://github.com/grpc/grpc/pull/5583#issuecomment-192134971",
    "issue_url": "https://api.github.com/repos/grpc/grpc/issues/5583",
    "id": 192134971,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE5MjEzNDk3MQ==",
    "user": {
      "login": "stanley-cheung",
      "id": 11674202,
      "node_id": "MDQ6VXNlcjExNjc0MjAy",
      "avatar_url": "https://avatars2.githubusercontent.com/u/11674202?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/stanley-cheung",
      "html_url": "https://github.com/stanley-cheung",
      "followers_url": "https://api.github.com/users/stanley-cheung/followers",
      "following_url": "https://api.github.com/users/stanley-cheung/following{/other_user}",
      "gists_url": "https://api.github.com/users/stanley-cheung/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/stanley-cheung/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/stanley-cheung/subscriptions",
      "organizations_url": "https://api.github.com/users/stanley-cheung/orgs",
      "repos_url": "https://api.github.com/users/stanley-cheung/repos",
      "events_url": "https://api.github.com/users/stanley-cheung/events{/privacy}",
      "received_events_url": "https://api.github.com/users/stanley-cheung/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-03-04T06:35:42Z",
    "updated_at": "2016-03-04T06:35:42Z",
    "author_association": "CONTRIBUTOR",
    "body": "The only test failure is a Python test, is it ok to merge?\n"
  },
  {
    "url": "https://api.github.com/repos/grpc/grpc/issues/comments/196383342",
    "html_url": "https://github.com/grpc/grpc/pull/5583#issuecomment-196383342",
    "issue_url": "https://api.github.com/repos/grpc/grpc/issues/5583",
    "id": 196383342,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE5NjM4MzM0Mg==",
    "user": {
      "login": "jtattermusch",
      "id": 9939684,
      "node_id": "MDQ6VXNlcjk5Mzk2ODQ=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/9939684?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/jtattermusch",
      "html_url": "https://github.com/jtattermusch",
      "followers_url": "https://api.github.com/users/jtattermusch/followers",
      "following_url": "https://api.github.com/users/jtattermusch/following{/other_user}",
      "gists_url": "https://api.github.com/users/jtattermusch/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/jtattermusch/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/jtattermusch/subscriptions",
      "organizations_url": "https://api.github.com/users/jtattermusch/orgs",
      "repos_url": "https://api.github.com/users/jtattermusch/repos",
      "events_url": "https://api.github.com/users/jtattermusch/events{/privacy}",
      "received_events_url": "https://api.github.com/users/jtattermusch/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-03-14T15:59:18Z",
    "updated_at": "2016-03-14T15:59:18Z",
    "author_association": "CONTRIBUTOR",
    "body": "Python test failure unrelated and win C failure unrelated as well (this is only touching PHP code). Merging this because it seems to be very low risk PR and we currently have a big merge queue after the fixit.\n"
  }
]
